subprojects/core/src/main/java/org/gradle/internal/classpath/InstrumentingTransformer.java

```diff
@@ -109,6 +111,15 @@
 
     private static final String[] NO_EXCEPTIONS = new String[0];
 
+    private static final List<Pair<String, String>> RENAMED_INTERFACES = asList(
```

Comment:
:x: we should not revert the changes to this type except the presence of `AbstractTask` in this list. We should keep the extracted `InstrumentingBackwardsCompatibilityVisitor.java`.

Reply:
Fixed.